### PR TITLE
[GEOS-7249] Add calculate from SRS bounds to Layer Group page

### PR DESCRIPTION
**Pull request by dvntucker**
From:

https://osgeo-org.atlassian.net/browse/GEOS-7249

Included new UI functionality, unit tests and documentation updates.

### CODE REVIEW

File: src/main/src/main/java/org/geoserver/catalog/LayerGroupHelper.java [hunk 1]
```diff
@@ -167,6 +168,24 @@ public void calculateBounds(CoordinateReferenceSystem crs) throws Exception {
...
+     * If the CRS has no bounds then the layer group bounds are set to null instead
+     * 
+     * @param crs
+     */
+    public void calculateBoundsFromCRS(CoordinateReferenceSystem crs) {
+        Envelope crsEnvelope = CRS.getEnvelope(crs);
+        if (crsEnvelope != null) {
+            ReferencedEnvelope refEnvelope = new ReferencedEnvelope(crsEnvelope);
+            this.group.setBounds(refEnvelope);
+        }
```
Comment (aaime) [inline src/main/src/main/java/org/geoserver/catalog/LayerGroupHelper.java]: This is not following the coding convetions

**Comment by aaime** [comment]
Looks good minus one rather visible breakage of coding conventions

**Comment by dvntucker** [comment]
Fixed the code style issue. Also update my IDE with the code style XML. Thought I had it already but I guess not.

File: src/web/core/src/main/java/org/geoserver/web/data/layergroup/AbstractLayerGroupPage.java [hunk 2]
```diff
@@ -166,6 +166,20 @@ public void onClick(AjaxRequestTarget target, Form form) {
             }
         });
         
+        form.add(new GeoServerAjaxFormLink("generateBoundsFromCRS"){
```
Comment (aaime) [inline src/web/core/src/main/java/org/geoserver/web/data/layergroup/AbstractLayerGroupPage.java]: Copyright year not updated

File: src/main/src/test/java/org/geoserver/catalog/LayerGroupHelperTest.java [hunk 3]
```diff
@@ -21,6 +21,8 @@
 import org.junit.Assert;
```
Comment (aaime) [inline src/main/src/test/java/org/geoserver/catalog/LayerGroupHelperTest.java]: Copyright header missing, could you add it?

File: src/web/core/src/test/java/org/geoserver/web/data/layergroup/LayerGroupEditPageTest.java [hunk 4]
```diff
@@ -5,7 +5,13 @@
  */
```
Comment (aaime) [inline src/web/core/src/test/java/org/geoserver/web/data/layergroup/LayerGroupEditPageTest.java]: Copyright year not updated

**Comment by aaime** [comment]
Thank you. Last time I missed that some of the copyright years and headers were not updated... sorry about that! @jodygarnett will be happier if we maintain the headers in proper state

**Comment by dvntucker** [comment]
Added/updated copyright notices on affected files.

**Comment by aaime** [comment]
Looking good, merged

**Pull request merged**